script.js

```diff
@@ -0,0 +1,30 @@
+//Declare and initilize variables
+let howOldAreYou;
+let invalidInput;
+let Age;
+let approvedAge = 21;
```

Comment:
Variables that do not change should always be a `const`
```suggestion
const approvedAge = 21;
```

Reply:
Unrelated, `minimumAge` is a more descriptive variable name